web/src/lib/components/AuctionView.svelte

```diff
@@ -89,7 +89,11 @@
 </script>
 
 <svelte:head>
-    <title>Auction</title>
+    {#if auction}
+        <MetaTag title="Auction" image={auction.media[0].url} description={"@"+ auction.seller.username + ": " + auction.title}/>
```

Comment:
I think the title should not be "Auction" but rather something based on the auction's actual title.

To make it even cleaner, maybe add a `title()` and a `shortDescription()` and a `image()` to IEntity, which you can implement in Auction and in Campaign (the title could be the auction's/campaign's title itself, but the `shortDescription` would be something like you do above - where you concatenate the username to some other text.. like "an auction by X"). Then simply pass the auction/campaign to MetaTag.svelte (which will receive an IEntity) and pick the title/short description/image in a uniform way.